conda_env/cli/main_create.py

```diff
@@ -51,22 +53,36 @@ def configure_parser(sub_parsers):
         '-q', '--quiet',
         default=False,
     )
+    p.add_argument(
+        'handle',
```

Comment:
two problems:

1) If I run:
` conda env create malev/my-project` then I'll have an environment named `my-project` or whatever environment's name I have on my environment.yml. What I am saying is: I define `malev/my-project` as name and then the environment will be named differently. I think it looks confusing.
2) Deprecating `--name` should be in another PR. This change will affect multiple parts of the code and documentation

makes sense?